merlinx.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "acex88",
+			"oem": "Redmi",
+			"device": "Redmi Note 9",
+			"filename": "crDroidAndroid-14.0-20231222-merlinx-v10.0-BETA.zip",
```

Comment:
don't push beta for new device
please rebuild with stable crDroid sources

Reply:
Okay 
I will do that as soon as possible